packages/terra-avatar/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ ChangeLog
 
 Unreleased
 ----------
+### Breaking Changes
+* `generic` subcomponent replaces the `sharedUser` subcomponent
```

Comment:
Can we add a rationale for this breaking change?  We're trying to make sure we provide that for breaking changes going forward.

Reply:
I have modified the change log for this code changes: 128dd373cceac974bb43230797125e826399df56 . Let me know if any changes needs to be made.